src/xdocs/config_filters.xml

```diff
@@ -775,10 +776,27 @@ public class UserService {
           You can use the command line helper tool to generate xpath
           suppressions based on your configuration file and input files.
           See
-          <a href="cmdline.html">here</a>
+          <a href="https://checkstyle.org/cmdline.html">here</a>
```

Comment:
@pbludov @romani Unrelated to these PRs, but what is your opinion on these changes?
Do we really need to embed domain in xdocs or should they stay relative?

I technically prefer relative in case we switch domains, but I also like that it will force you to the latest domain if there are multiple working ones, like there is right now.

Reply:
Relative links are broken on the javadoc side. For example:
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.html
There is two broken links on this page: `IllegalInstantiation`, `IllegalImport`

On this page: https://checkstyle.org/config_coding.html#IllegalType they works as expecting.

Reply:
I am talking xdoc only. On javadoc side, like you said, we must have a full URL otherwise the URL won't work.

Reply:
We should keep relative links in xdocs, but convert them to absolute in javadocs.